src/components/Board/Navbar/Navbar.css

```diff
@@ -60,3 +60,19 @@
 .Navbar .LoginSignUpButton:focus {
   color: #fff;
 }
+
+@media (max-width: 768px) {
+  .Navbar__group--end .MuiIconButton-root,
+  .Navbar__group--start .MuiIconButton-root {
```

Comment:
☘️  Is generally better to avoid using Material UI classnames to style the components.
Is better to use a custom class name like `Navbar__group--button`.
Also i noticed that using `Navbar__group--end` and `Navbar__group--start` is not necessary.
You can use directly `.Navbar__group .MuiIconButton-root` for your actual code. (or `.Navbar__group .Navbar__group--button` taking this comment)